### PR TITLE
Intro page reworks

### PR DESCRIPTION
**Pull request by a1ttech**
- new intro pages for overview, access control, PKPs, and Lit Actions
- more projects added to 'projects building with Lit'
- cleaned up use cases page and added more examples

### CODE REVIEW

File: docs/Ecosystem/projects.md [hunk 1]
```diff
@@ -6,23 +6,70 @@
...
+- **[P00ls](https://www.p00ls.io/)**: A web3 platform for creators and brands.
 - **[Gateway](https://www.mygateway.xyz/home)**: Verifiable credential platform.
 - **[Tempra](https://ethglobal.com/showcase/tempra-xnxf4)**: Decentralized social platform inspired by Snapchat.
+- **[Lens](https://lens.xyz/)**: Token-gated content on Lens.
+- **[The 402](https://app.the402.xyz/)**: Live-streaming and NFT marketplace.
+- **[MemeDAO](https://github.com/Eyon42/MemeDAO)**: DAO communication platform that uses Lit to support private, members-only content.
+- **[Iris](https://github.com/irisxyz/iris)**: A decentralized social media and subscription platform built on Lens.
+- **[DecentraScholar](https://github.com/TanusreeSharma/LFGrow-hack)**: A platform that incentivizes collaboration among academic researchers.
+- **[MxTape](https://ethglobal.com/showcase/mxtape-1tr3a)**: Decentralized music player.
+- **[Peal](https://github.com/Pearl-Market/marketplace)**: Discover and share typefaces and designs on-chain.
```
Comment (debbly) [inline docs/Ecosystem/projects.md:23]: mispelling, should be Pearl?

File: docs/Ecosystem/projects.md [hunk 2]
```diff
@@ -6,23 +6,70 @@
...
+- **[AQA Protocol](https://ethglobal.com/showcase/aqa-protocol-b5ok4?ref=spark.litprotocol.com)**: Give users full control over their private data.
+- **[VerifyXYZ](https://github.com/verify-xyz/did-profiles/tree/poc-lit?ref=spark.litprotocol.com)**: Essential building blocks for Decentralized ID.
+
+
+### Privacy and Encryption
+
+- **[dKeeper](https://ethglobal.com/showcase/dkeeper-3ma7r?ref=spark.litprotocol.com)**: Access control management for dApps.
+- **[Selfguard](https://github.com/selfguard/selfguard?ref=spark.litprotocol.com)**: Universal API for encryption.
+- **[Starling Lab](https://github.com/starlinglab/archive-explorer/)**: Token-gated data archives.
+- **[Curve Labs](https://github.com/Curve-Labs/lit-privacy/tree/main)**: Privacy-preserving infrastructure tooling.
```
Comment (debbly) [inline docs/Ecosystem/projects.md:54]: A better description for the curve labs project is, "Tooling for private on-chain voting"

File: docs/LitActions/intro.md [hunk 3]
```diff
@@ -12,61 +12,35 @@ To start developing with Lit Actions, check out examples [here](/LitActions/hell
...
 
-## What can I use Lit Actions for?
+Every Lit Action gets executed across Lit’s threshold cryptography network in parallel, meaning the result of each program is independently verified by each node. Once a threshold of nodes have verified the result (more than two-thirds of network participants), the signing or decryption logic defined therein can be executed.
 
-When used in conjunction with PKPs, Lit Actions facilitate automated signing and decryption of data, functionally combining smart contracts with an individual key-pair.
+A trivial example would be a Lit Action and associated PKP that checks if a number is prime, only returning a signature it if it is prime. Each node will execute the Lit Action with a submitted input and verify that it meets the required conditions. If it does, the node will provision an independent key share. Only after more than two-thirds of these shares have been collected can the complete signature be formed.
```
Comment (sarahzdev) [inline docs/LitActions/intro.md:31]:
```suggestion
A trivial example would be a Lit Action and associated PKP that checks if a number is prime, only returning a signature if the number is prime. Each node will execute the Lit Action with a submitted input and verify that it meets the required conditions. If it does, the node will provision an independent key share. Only after more than two-thirds of these shares have been collected can the complete signature be formed.
```

**Comment by sarahzdev** [comment]
anyone remember why there may be duplicate files? e.g., `docs/accessControl/intro.md` and `docs/coreConcepts/accessControl/intro.md`

**Comment by debbly** [comment]
> anyone remember why there may be duplicate files? e.g., `docs/accessControl/intro.md` and `docs/coreConcepts/accessControl/intro.md`

Yes - generally it's a linux system that's case sensitive vs these links get resolved on mac systems, but those two links aren't the same since one is `coreConcepts` ?

File: docs/coreConcepts/accessControl/intro.md
Comment (debbly) [inline docs/coreConcepts/accessControl/intro.md:1]: If this is a duplicate of the other access control doc, please delete this and check the `sidebars.js` has the correct path to the accessControl intro left

**Review by debbly — APPROVED** [review]

**Pull request merged**